mubench.pipeline/data/build_command.py

```diff
@@ -0,0 +1,190 @@
+import logging
+import shutil
+import os
+import shlex
+from glob import glob
+from os import makedirs
+from os.path import join, exists, dirname, splitext, relpath
+from typing import List, Set
+
+from utils.io import remove_tree, copy_tree
+from utils.shell import Shell, CommandFailedError
+
+
+class BuildCommand:
+    def __init__(self, name: str, args: List[str]):
+        self.logger = logging.getLogger("buildcommand")
```

Comment:
I would prefer to pass the logger into `execute` from the caller, because this is consistent with how `Shell.exec` works and it allows the caller to care about things like the correct nesting depth (currently, the output would not be indented, whereas before it was indented by two levels).